### PR TITLE
Removed unnecessary parenthesis detail about an abbreviation for candidate

### PR DESCRIPTION
**Pull request by audreyandoy**
"(short for candidate)" doesn't seem necessary? I could be wrong

### CODE REVIEW

File: functions-continued/problem-set-functions-continued.md [hunk 1]
```diff
@@ -250,7 +250,7 @@ def get_third_place(finalists):
 * title: Functions, continued
 ### !question
 
-Create a function named `compare_votes`. It has two parameters, `candidate_a` and `candidate_b` (short for "candidate"). Both `candidate_a` and `candidate_b` will be dictionaries that look like this:
+Create a function named `compare_votes`. It has two parameters, `candidate_a` and `candidate_b`. Both `candidate_a` and `candidate_b` will be dictionaries that look like this:
```
Comment (anselrognlie) [inline functions-continued/problem-set-functions-continued.md:253]: This also affects the next question (line 306) (I was tracking this in the typo log!).  Can you fix it there as well?

**Review by anselrognlie — CHANGES_REQUESTED** [review]
Both question 6 and 7 have this issue. Can you fix it in both?

File: functions-continued/problem-set-functions-continued.md (hunk 1 above)
Comment (audreyandoy) [reply functions-continued/problem-set-functions-continued.md:253]: sure thing!

**Review by anselrognlie — APPROVED** [review]
Looks good!

**Pull request merged**